examples/Utils.cpp

```diff
@@ -262,8 +262,8 @@ void CreateDefaultRenderPass(const nxt::Device& device, nxt::RenderPass* renderP
 
 nxt::Buffer CreateFrozenBufferFromData(const nxt::Device& device, const void* data, uint32_t size, nxt::BufferUsageBit usage) {
     nxt::Buffer buffer = device.CreateBufferBuilder()
-        .SetAllowedUsage(nxt::BufferUsageBit::MapWrite | usage)
```

Comment:
Can you do this changes in a separate commit?
Also there are a bunch of places that were not changed to account for this (for example glTFViewer.cpp). ```git grep MapWrite``` will give a superset of the places where this is needed.